internal/runner/tools/terragrunt/terragrunt.go

```diff
@@ -2,38 +2,51 @@ package terragrunt
 
 import (
 	"errors"
-	"os"
 	"os/exec"
 
+	"github.com/padok-team/burrito/internal/runner/tools/opentofu"
 	"github.com/padok-team/burrito/internal/runner/tools/terraform"
+	c "github.com/padok-team/burrito/internal/utils/cmd"
 )
 
 type Terragrunt struct {
 	ExecPath   string
 	WorkingDir string
 	Terraform  *terraform.Terraform
+	OpenTofu   *opentofu.OpenTofu
```

Comment:
I think we can abstract the tool used to have only one property. For Terragrunt, we only need the execution path of the tool.

I see 2 solutions:
1. We add another function to the `TerraformExec` interface to obtain the execution path of the tool
2. We just give a string `BaseToolPath` to the `Terragrunt` struct instead of a pointer to an object